cu_digital_campaign_bundle.module

```diff
@@ -13,153 +28,44 @@
-    'administrator' => array(
-      'access all webform results',
-      'access own webform results',
-      'access own webform submissions',
-      'administer cu feedback module',
-      'create webform content',
-      'delete all webform submissions',
-      'delete any webform content',
-      'delete own webform content',
-      'delete own webform submissions',
-      'edit all webform submissions',
-      'edit any webform content',
-      'edit own webform content',
-      'edit own webform submissions',
-      'edit webform components',
-    ),
-    'anonymous user' => array(),
-    'authenticated user' => array(
-      'delete own webform submissions',
-      'edit own webform submissions',
-    ),
-    'content_editor' => array(
-      'access own webform results',
-      'create webform content',
-      'delete any webform content',
-      'delete own webform content',
-      'edit any webform content',
-      'edit own webform content',
-      'edit webform components',
+    'campaign_manager' => array(
+      'administer embed templates',
+      'delete embeds',
+      'edit et_slate_form_bean embeds',
+      'edit et_slate_form_block embeds',
+      'publish embeds',
+      'unpublish embeds',
     ),
     'developer' => array(
-      'access all webform results',
-      'access own webform results',
-      'access own webform submissions',
-      'administer cu feedback module',
-      'create webform content',
-      'delete all webform submissions',
-      'delete any webform content',
-      'delete own webform content',
-      'delete own webform submissions',
-      'edit all webform submissions',
-      'edit any webform content',
-      'edit own webform content',
-      'edit own webform submissions',
-      'edit webform components',
+      'administer embed templates',
+      'delete embeds',
+      'edit et_slate_form_bean embeds',
+      'edit et_slate_form_block embeds',
+      'publish embeds',
+      'unpublish embeds',
     ),
-    'edit_my_content' => array(),
     'site_owner' => array(
-      'access all webform results',
-      'access own webform results',
-      'access own webform submissions',
-      'administer cu feedback module',
-      'create webform content',
-      'delete all webform submissions',
-      'delete any webform content',
-      'delete own webform content',
-      'delete own webform submissions',
-      'edit all webform submissions',
-      'edit any webform content',
-      'edit own webform content',
-      'edit own webform submissions',
-      'edit webform components',
+      'administer embed templates',
+      'delete embeds',
+      'edit et_slate_form_bean embeds',
+      'edit et_slate_form_block embeds',
+      'publish embeds',
+      'unpublish embeds',
+    ),
+    'anonymous user' => array(
+      'view et_slate_form_bean embeds',
+      'view et_slate_form_block embeds',
+    ),
+    'authenticated user' => array(
+      'view et_slate_form_bean embeds',
```

Comment:
I think we usually explicitly put all permissions that a role has instead of implicitly using the authenticated user inheritance. For example, in express_permissions "view any block bean" is included in administrator, anonymous, and authenticated user roles.